music21/analysis/neoRiemannian.py

```
@@ -391,9 +423,158 @@ def hexatonicSystem(c):
+    >>> [x.name for x in UFMCmaj.pitches]
+    ['E-', 'G', 'B-']
+    >>> USMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='USM')
+    >>> [x.name for x in USMCmaj.pitches]
+    ['E', 'G#', 'B']
+    >>> LFMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='LFM')
+    >>> [x.name for x in LFMCmaj.pitches]
+    ['A-', 'C', 'E-']
+    >>> LSMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='LSM')
+    >>> [x.name for x in LSMCmaj.pitches]
+    ['A', 'C#', 'E']
+    '''
+
+    if raiseException is True:
+        options = ['UFM', 'USM', 'LFM', 'LSM']
+        if transformation not in options:
+            raise NeoR_Exception('Transformation must be one of %s' %options)
+
+    if c.forteClassTnI == '3-11': # if (c.isMajorTriad() or chord.isMinorTriad()):
+        if transformation == 'UFM':
+            newChord = c.transpose(3)
+        elif transformation == 'USM':
+            newChord = c.transpose(4)
+        elif transformation == 'LFM':
+            newChord = c.transpose(-4)
+        elif transformation == 'LSM':
+            newChord = c.transpose(-3)
+
+    outChord = _simplerEnharmonics(newChord)
+    outChord.quarterLength = c.quarterLength
+
+    return outChord
+
+def disjunctMediants(c, UpperOrLower='Upper', raiseException=True):
+    '''
+    Transforms a chord into the upper or lower disjunct mediant.
+    These transformations involve:
+    root motion by a non-diatonic third;
+    mode-change;
+    no common-tones.
+
+    >>> Cmaj = chord.Chord('C5 E5 G5')
+    >>> UpChrom = analysis.neoRiemannian.disjunctMediants(Cmaj, UpperOrLower='Upper')
+    >>> [x.name for x in UpChrom.pitches]
+    ['E-', 'G-', 'B-']
+    >>> DownChrom = analysis.neoRiemannian.disjunctMediants(Cmaj, UpperOrLower='Lower')
+    >>> [x.name for x in DownChrom.pitches]
+    ['A-', 'C-', 'E-']
+    '''
+
+    if raiseException is True:
```

raiseException is unnecessary -- if the keyword is wrong it should raise an exception no matter what.  raiseException type work is only used when the data might be corrupt or not.